feature-libs/order/core/store/effects/order-details.effect.spec.ts

```diff
@@ -40,6 +41,14 @@ class MockUserIdService implements Partial<UserIdService> {
   }
 }
 
+class MockLoggerService {
```

Comment:
as discussed, this mock class could be imported to avoid repetition. 
suggestion to put it in  cypress/helpers folder, new file logger.ts